### ISSUE TITLE
Diskussion

### ISSUE THREAD

**Issue by jonasmolsbeck** [fynn-oldenburg/GitHub-Projekt-Wissenschaftliches-Arbeiten#65]
Hier können wir die Diskussion führen, sobald die Analyse fertig ist.

**Comment by fynn-oldenburg** [comment]
Die Analyse einiger Kategorien hat folgendes ergeben:

```
## Kategorisierung ordinaler Variablen

categorize_ordinal(data, by = 'Mathe', bins = 3, in_place=TRUE) %>%
    group_by(Mathe) %>%
    summarize(n = sum(MatheLK == 'ja'))
# Wenn niedriges Interesse an Mathe vorliegt, dann wird eher nicht der MatheLK gewählt

categorize_ordinal(data, by = 'Mathe', bins = 3, in_place=TRUE) %>%
    group_by(Mathe) %>%
    summarize(n = sum(Studienfach == 'Mathe'))
# Wenn niedriges Interesse an Mathe vorliegt, dann wird gar nicht das Studienfach Mathe gewählt?

categorize_ordinal(data, by = 'Programmieren', bins = 3, in_place=TRUE) %>%
    group_by(Programmieren) %>%
    summarize(n = sum(Studienfach == 'Informatik'))
# bei höherem Interesse am Programmieren wird eher der Studiengang Informatik gewählt

categorize_ordinal(data, by = 'Programmieren', bins = 3, in_place=TRUE) %>%
    group_by(Programmieren) %>%
    summarize(n = sum(Studienfach == 'Data Science'))
# Interesse an Programmieren scheint einen positiven Einfluss auf die Wahl von Data Science als Studienfach zu haben.

categorize_ordinal(data, by = 'Programmieren', bins = 3, in_place=TRUE) %>%
    group_by(Programmieren) %>%
    summarize(n = sum(Studienfach == 'Statistik'))
# Interesse am Programmieren ist nicht ausschlaggebend für die Wahl des Statistik-Studiums.
# Eher ein gegenteiliger Effekt ist zu vermerken. Vermutlich wählen die Personen dann eher Data Science?
```

@jonasmolsbeck @danfleon stimmen die vermuteten Zusammenhänge mit der Simulation überein?

**Comment by jonasmolsbeck** [comment]
_Wenn niedriges Interesse an Mathe vorliegt, dann wird eher nicht der MatheLK gewählt_ 
-> Korrekt, allgemein erhöht ein hohes Mathe-Interesse  die Wahrscheinlichkeit, dass  die Person im Mathe LK war

**Comment by jonasmolsbeck** [comment]
Die genaue Wahrscheinlichkeiten geben wir zum Schluss bekannt @danfleon und ich habe schon etwas dafür vorbereitet

**Comment by fynn-oldenburg** [comment]
Ist es richtig, dass ein niedriges Mathe-Interesse dazu führt, dass niemals Mathe als Studienfach gewählt wird? Also gibt es da einen cut-off Punkt?

**Comment by fynn-oldenburg** [comment]
Oder liegt das nur an der Stichprobe, dass in der Kategorie niemand Mathe studiert?

**Comment by jonasmolsbeck** [comment]
> Ist es richtig, dass ein niedriges Mathe-Interesse dazu führt, dass niemals Mathe als Studienfach gewählt wird? Also gibt es da einen cut-off Punkt?
Ja, genau. Die Wkeit, dass ein Mathestudent ein Matheinteresse >=5 hat beträgt 98%

**Comment by jonasmolsbeck** [comment]
_bei höherem Interesse am Programmieren wird eher der Studiengang Informatik gewählt_
_Interesse an Programmieren scheint einen positiven Einfluss auf die Wahl von Data Science als Studienfach zu_
Auch das ist richtig, Die Studierenden der Studiengänge Informatik und Data Science haben eine deutlich höhere Wahrscheinlichkeit, dass sie ein großes bzw. größeres Interesse an Programmieren haben.

**Comment by danfleon** [comment]
@fynn-oldenburg, meiner Meinung nach waren die Ergebnisse deiner Analyse einiger Kategorien schon in der Richtung für die gewählten Wahrscheinlichkeiten der Interesse pro Studienfach. Es sieht gut aus.

**Comment by WlanNergady** [comment]
Anwendung der Funktion stats_bivariate_metric_dichotom ergibt, dass zwischen Alter und Mathe-LK kein großer Unterschied zwischen den Gruppen "ja" und "nein", also kein Zusammenhang erkennbar ist (was auch der 
Aufgabenstellung entspricht)

**Comment by jonasmolsbeck** [comment]
> Anwendung der Funktion stats_bivariate_metric_dichotom ergibt, dass zwischen Alter und Mathe-LK kein großer Unterschied zwischen den Gruppen "ja" und "nein", also kein Zusammenhang erkennbar ist (was auch der Aufgabenstellung entspricht)

Das Alter haben wir tatsächlich in der Simulation für die anderen Variablen nicht miteinbezogen.

**Comment by PhilippSL** [comment]
# Wenn niedriges Interesse an Mathe vorliegt, dann wird eher nicht der MatheLK gewählt
# Wenn niedriges Interesse an Mathe vorliegt, dann wird gar nicht das Studienfach Mathe gewählt?

durch den Aufruf von
bivariate_stats_categorical(data, "Mathe", "MatheLK") ergibt sich ein Cramer's V von ca. 0,77.

Ein Cramer's V von über 0.7 spricht für einen starken Zusammenhang 
von dem Interesse an Mathe und der Wahl für oder gegen den Mathe-Leistungskurs.


https://github.com/fynn-oldenburg/GitHub-Projekt-Wissenschaftliches-Arbeiten/blob/3acbad8c0270d5d9d4c5f066ae9e05e711bb2102/analysis/analysis.R#L90-L93

**Comment by jonasmolsbeck** [comment]
@fynn-oldenburg @PhilippSL  @PhilippSL  Ihr findet jetzt auch in der **readme** eine Übersicht der Sample-Wahrscheinlichkeiten, die wir für die Simulation verwendet haben.

**Comment by danfleon** [comment]
Warum wurde den Mathe-Leistungskurs eine Role spielen? Gibt es Vermutungen? (Persönlich habe ich keine Ahnung, weil ich das nicht kenne)

**Comment by WlanNergady** [comment]
Es ergibt Sinn, dass die Wahl des Mathe-LK die Wahl des Studienfachs Mathe begünstigt (bzw. umgekehrt), da tendenziell die Schüler im Mathe-LK gut in Mathe sind bzw. ein gewisses Interesse daran haben (und dann auch Mathe als Studienfach wählen).

**Comment by PhilippSL** [comment]
Die erste Übersicht zeigte: 
```
## Ein großer Teil der Personen studiert Statistik, 
## ein besonders geringer Teil der Personen studiert Mathe
## Ein Großteil (70%) der Befragten hat einen Mathe-Leistungskurs belegt
```

**Comment by jonasmolsbeck** [comment]
> Ein großer Teil der Personen studiert Statistik
Das liegt an dem sample, theoretisch haben Statistik und Data Science beide eine Wahrscheinlichkeit von 35%

**Comment by WlanNergady** [comment]
Zur Simulation vom Mathe-LK finde ich interessant, dass ihr das Interesse am Programmieren als Faktor einbezogen habt (es stand in der Aufgabenstellung, aber war ja als optional angegeben)... hätte ich wahrscheinlich nicht gemacht

**Comment by jonasmolsbeck** [comment]
> Ein besonders geringer Teil der Personen studiert Mathe
Das Studienfach Mathe hatte in unserer Simulation eine Wkeit von 10%

**Comment by WlanNergady** [comment]
Ich habe auch gesehen, dass das Interesse am Programmieren immerhin nicht so einen großen Einfluss auf die Wahl des Mathe-LKs hat... Warum habt ihr da als Cutoff-Wert eigentlich >4 gewählt?

**Comment by danfleon** [comment]
@WlanNergady, weil >4 ist überdurchschnittlich.

**Comment by jonasmolsbeck** [comment]
> Ein Großteil (70%) der Befragten hat einen Mathe-Leistungskurs belegt.
Das liegt daran, dass Mathe-LK mit dem Interesse an Mathematik und Programmieren zusammenhängt und das Interesse an Mathe und Programmieren im allgemeinen für alle relativ groß ist. Zudem haben wir insbesondere für Mathe-Studierende eine hohe Grund-Wahrscheinlichkeit für Mathe LK.
